qtconsole/usage.py

```diff
@@ -0,0 +1,221 @@
+"""
+Usage information for QtConsole
+"""
+
+# Copyright (c) Jupyter Development Team.
+# Distributed under the terms of the Modified BSD License.
+
+
+def page_guiref(arg_s=None):
+    """Show a basic reference about QtConsole."""
+    from IPython.core import page
+    page.page(gui_reference)
+
+
+gui_reference = """\
+=====================
+The Jupyter QtConsole
+=====================
+
+This console is designed to emulate the look, feel and workflow of a terminal
+environment, while adding a number of enhancements that are simply not possible
+in a real terminal, such as inline syntax highlighting, true multiline editing,
```

Comment:
We might want to change the phrasing, because the first two of these (syntax highlighting and multiline editing) are coming to IPython for version 5 (see ipython/ipython#9118). And even inline graphics are [possible](https://github.com/saitoha/PySixel), albeit not supported by most terminal emulators.